src/documents/consumer.py

```diff
@@ -779,6 +789,13 @@ def apply_overrides(self, document):
             }
             set_permissions_for_object(permissions=permissions, object=document)
 
+        if self.override_custom_field_ids:
+            for field_id in self.override_custom_field_ids:
+                field = CustomField.objects.get(pk=field_id)
+                document.custom_fields.add(
```

Comment:
I don't think you need the `.add` here

Reply:
Sorry if im missing something obvious, why not (or what do you mean instead)? This is adding an instance for each

Reply:
The custom_fields will be update when a CustomFieldInstance is created linked to that document.  So I don't think you need to do anything with the field on the Document.

Reply:
ahh, yes, duh. Thanks, will fix